Plugins/GitHub3/GitHubCredentialsPrompt.Designer.cs

```diff
@@ -179,6 +180,7 @@ private void InitializeComponent()
             this.ClientSize = new System.Drawing.Size(409, 206);
             this.Controls.Add(this.btn_OK);
             this.Controls.Add(this.tableLayoutPanel1);
+            this.Icon = ((System.Drawing.Icon)(resources.GetObject("$this.Icon")));
```

Comment:
This would duplicate the icon in an anonymous way.
The icon is not available in the plugin interface, @ivangrek something to add?
The icon could be taken from the active form instead
```suggestion
            this.Icon = ResourceManager.GitExtensionsFormBase.ActiveForm.Icon;
```

Reply:
Good idea, i have updated the PR.

Reply:
Just use GitExtensionsFormBase as base form like in other cases.

![image](https://user-images.githubusercontent.com/3169265/103038098-3ace7b00-4576-11eb-8695-a94802f36a85.png)